core/src/main/java/tc/oc/pgm/PGMConfig.java

```diff
@@ -183,6 +184,7 @@ public final class PGMConfig implements Config {
     this.showProximity = parseBoolean(config.getString("ui.proximity", "false"));
     this.showSideBar = parseBoolean(config.getString("ui.sidebar", "true"));
     this.showTabList = parseBoolean(config.getString("ui.tablist", "true"));
+    this.resizeTabList = parseBoolean(config.getString("ui.resize", "false"));
```

Comment:
```suggestion
    this.resizeTabList = parseBoolean(config.getString("ui.tablist-resize", "false"));
```
i think "ui.resize" is just too vague, resize what part of the ui?